packages/webapp/components/invite/Referral.tsx

```diff
@@ -0,0 +1,181 @@
+          </p>
+        </span>
+        <h1 className="w-full text-center laptop:text-left break-words-overflow typo-large-title tablet:typo-mega2 laptopL:typo-giga2">
+          The homepage
+          <span className="block font-bold text-theme-color-cabbage">
+            {' '}
+            developers deserve
+          </span>
+        </h1>
+        <p className="mx-auto laptop:mx-0 mt-7 tablet:mt-8 max-w-sm laptopL:max-w-xl text-center laptop:text-left typo-title3 text-theme-label-tertiary">
+          Get a feed of the best developer news out there! Read more quality
+          articles.
+          <span className="tablet:block laptopL:inline">Stay up to date.</span>
+        </p>
+        <Button
+          buttonSize={ButtonSize.Large}
+          // important has been used is some classnames to override the default styles as agreed on [thread](https://dailydotdev.slack.com/archives/C05P9ET7S9K/p1699023366663489?thread_ts=1699011522.350609&cid=C05P9ET7S9K)
+          className="p-4 mx-auto laptop:mx-0 mt-6 tablet:mt-12 max-w-[17.5rem] mobileL:max-w-[fit-content] btn-primary mobileL:!h-[4rem] mobileL:!p-[0] mobileL:!px-[1.5rem]"
+          tag="a"
+          href={downloadBrowserExtension}
+          onClick={() => {
+            handleAcceptClick();
+          }}
+          target="_blank"
+          rel={anchorDefaultRel}
+          loading={isLoading}
+          disabled={isLoading || isSuccess}
+        >
+          <FlexCentered className="gap-2 mobileL:typo-title3">
+            <BrowsersIcon className="text-theme-label-primary w-[3.7rem] h-[1.9rem] mobileL:w-[5rem] mobileL:h-[2.5rem]" />
+            Try it now - It&apos;s free
+          </FlexCentered>
+        </Button>
+      </div>
+      <div className="flex fixed -bottom-1/2 flex-auto w-full h-full laptop:initial laptop:bottom-[unset]">
+        <div className="flex relative laptop:absolute laptopL:-right-24 justify-center self-center m-auto mx-2 z-10 laptop:right-[-7.5rem] w-fit laptop:w-[initial]">
+          <img
+            src={
+              cloudinary.referralCampaign.genericReferral.purpleEdgeGlowTablet
+            }
+            alt="Purple glow right edge"
+            className="fixed laptop:top-0 -bottom-4 w-full laptop:w-auto laptop:h-full laptop:bottom-[unset] laptop:rotate-[270deg] z-20"
+          />
+          <img
+            src={cloudinary.referralCampaign.genericReferral.appScreenshot}
+            alt="Daily.dev app screenshot of my feed page"
+            className="object-contain w-full laptop:w-auto h-auto laptop:h-full z-10 laptopXL:max-h-[39.5rem] laptop:max-h-[25.35rem]"
+          />
+          <img
+            src={cloudinary.referralCampaign.genericReferral.playButton}
+            alt="Play daily.dev introduction video"
+            className="absolute laptop:self-center laptopL:m-auto laptopXL:h-auto laptop:mr-[8.5rem] laptop:mt-[0.5rem] z-20 laptop:h-[8rem] tablet:h-[14.5rem] h-[7.8rem] tablet:mt-[-1.5rem]"
```

Comment:
Let's keep using the pre-defined values.
```suggestion
            className="absolute laptop:self-center laptopL:m-auto laptopXL:h-auto laptop:mr-[8.5rem] laptop:mt-2 z-20 laptop:h-32 tablet:h-[14.5rem] h-[7.5rem] tablet:-mt-6"
```